include/controller/c/webots/vehicle/driver.h

```diff
@@ -35,9 +35,11 @@ typedef enum { UNDEFINED_CONTROL_MODE = -1, SPEED = 0, TORQUE } WbuDriverControl
 
 typedef enum { DOWN, SLOW, NORMAL, FAST } WbuDriverWiperMode;
 
+// private function for webots_ros2 to identify robots that can use libdriver
+bool initialization_is_possible();
```

Comment:
I would keep the prefix to avoid name clashes:
```suggestion
bool wbu_driver_initialization_is_possible();
```